content/en/blog/2023/otel-end-user-discussions-february-2023.md

```diff
@@ -0,0 +1,187 @@
+
+**Q:** Is there a way to sample traces based on the span counts? Example:
+Drop/truncate traces which have more than 1000 spans in a single trace.
+
+**More context:** Sometimes, due to issues in the application itself, some
+traces generate a lot of spans that are not needed. Is there a way in
+OpenTelemetry to control this? More specifically, is there a way in which we can
+set up a condition where if a certain trace has more than ‘n’ number of spans,
+we can drop or truncate the number of spans.
+
+**A:** Tail-based sampling processor provides users with a bunch of sampling
+policies. Span count is one such policy. You can also combine multiple policies.
+Here's the link to tail
+[sampling processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md).
+The span count policy is based on min span count. Some users might look for some
+kind of exclusion policy.
+
+#### 2 - Use cases of span links
+
+**Q:** What are the use cases of span links?
+
+**A:** Span links are used to have an overview when there are thousands of
+spans. It helps in navigation, and answers questions like, "Where does a trace
+start and end?"
+
+### OpenTelemetry Metrics
+
+#### 1 - Supporting other metrics format
+
+**Q:** Can Otel collector support metrics generated from other libraries like
+statsd library?
+
+**A:** The OpenTelemetry Collector contrib has a lot of receivers for different
+types of metrics that can be used. For example, if you are sending out metrics
+in Prometheus format, you can configure your Otel collector to scrape Prometheus
+metrics. There is also a statsd receiver that is available. If you have
+something that is already working, then you don’t need to change it. You can
+check the list of receivers
+[here](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver).
+
+#### 2 - Emitting business metrics
+
+**Q:** What signals are you using to emit business metrics? For instance, at an
+arbitrary point in time, emit something that resembles a counter but only emit
+it once.
+
+**A:** There is a
+[current issue](https://github.com/open-telemetry/opentelemetry-specification/issues/2318)
+regarding this which you can track.
+
+### OpenTelemetry Adoption & Enablement
+
+#### 1 - Improving contributions from APAC region
+
+**Q:** How do we improve contributions from APAC region?
+
+**A:** Suggestions from the community:
+
+- Reach out to current OpenTelemetry maintainers and share the challenges
+- Create a list of maintainers from APAC region to whom people can reach to
+- Local in-person meetups for OpenTelemetry users
+- A good place to start would be `good first issues` in any of the OTel repos,
+  and ask for help in GitHub issues
+- Join
+  [OTel slack community](https://communityinviter.com/apps/cloud-native/cncf)
+  and ping in relevant channels
+
+## Other Important discussion points
+
+The community also discussed these important points:
```

Comment:
It would be helpful to provide a bit of additional context for these points

Reply:
@avillela, thanks for the review. I have added the details.